src/main/java/mate/hwdao/dao/impl/jdbc/ManufactureDaoJdbcImpl.java

```diff
@@ -0,0 +1,124 @@
+                manufacturer.setCountry(resultSet.getObject(3, String.class));
+            }
+            resultSet.close();
+            preparedStatement.close();
+            return Optional.ofNullable(manufacturer);
+        } catch (SQLException ex) {
+            System.out.println("Connection failed..." + ex);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        Manufacturer manufacturer = null;
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        String query = "SELECT * FROM manufacture WHERE isexist = true";
+        try {
+            PreparedStatement preparedStatement
+                    = ConnectionUtil.getConnection().prepareStatement(query);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturer = new Manufacturer();
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+                manufacturer.setName(resultSet.getObject(2, String.class));
+                manufacturer.setCountry(resultSet.getObject(3, String.class));
+                manufacturers.add(manufacturer);
+            }
+            resultSet.close();
+            preparedStatement.close();
+        } catch (SQLException ex) {
+            System.out.println("Connection failed..." + ex);
+        }
+        return manufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String query = "UPDATE manufacture SET name = ?, country = ?"
+                + " WHERE id = ? AND isexist = true";
+        try {
+            PreparedStatement preparedStatement
+                    = ConnectionUtil.getConnection().prepareStatement(query);
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.setLong(3, manufacturer.getId());
+            if (preparedStatement.executeUpdate() > 0) {
+                preparedStatement.close();
+                return manufacturer;
+            }
+            preparedStatement.close();
+        } catch (SQLException ex) {
+            System.out.println("Connection failed..." + ex);
+        }
+        throw new RuntimeException("Can't find manufacturer with id " + manufacturer.getId());
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE manufacture SET isexist = false WHERE id = ?";
+        try (PreparedStatement preparedStatement
+                     = ConnectionUtil.getConnection().prepareStatement(query)) {
+            preparedStatement.setLong(1, id);
+            if (preparedStatement.executeUpdate() > 0) {
+                preparedStatement.close();
+                return true;
+            }
+        } catch (SQLException ex) {
+            System.out.println("Connection failed..." + ex);
+        }
+        return false;
+    }
```

Comment:
Don’t return true all the time in the method delete. Let’s return boolean value depending on preparedStatement.executeUpdate() result.